bb-new.html

```diff
@@ -0,0 +1,304 @@
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#partners">Partners</a>
+                  <div class="dropdown-content">
+                    <a href="#financial">Financial Services</a>
+                    <a href="#healthcare">Healthcare</a></li>
+                    <a href="#professional">Professional Services</a>
+                    <a href="#education">Education</a>
+                    <a href="#government">Government</a>
+                    <a href="#utilties">Utilities & Manufactoring</a>
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#partners">Support</a>
+                  <div class="dropdown-content">
+                    <a href="#products">Support by Products</a>
+                    <a href="#smartphone">Smartphones & Apps</a></li>
+                    <a href="#information">Products Inforamtion</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#blog">Blog</a>
+                  <div class="dropdown-content">
+                    <a href="#enterprise">Enterprise Blog</a>
+                    <a href="#developer">Developer</a></li>
+                    <a href="#support">Support Blog</a>
+                  </div>
+              </div>              
+              <div class="dropdown topnav-right">
+                <a href="#contact">Telephone icon</a>
+                  <div class="dropdown-content">
+                    <a href="#register">Register for updates</a>
+                    <a href="#sales">Contact Sales</a></li>
+                    <a href="#contactus">Contact Us</a>
+                  </div>
+                </div> 
+              <div class="topnav-right">
+                <a href="#search">Search icon</a>
+              </div>           
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="column">
+            <img src="./images/windmill.jpg" alt="several windmills in a line">
+            <h1 class="hero-text">OUR DEVICES CHANGED WHERE WE WORK.
+            <br/>
+              OUR SECURITY IS CHANGING HOW WE WORK.
+            </h1>
+            <br/>
+            <br/>
+            <br/>    
+            <p class="hero-text text-content">From cybersecurity, to critical event management, to the internet of Things, when we say intelligent secuirty everywhere- we mean everywhere</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <div class="btn hero-text">
+            <button>WATCH VIDEO</button>
+          </div>        
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-2">
+          <div class="divider">
```

Comment:
Within this div, you are styling each individual element it seems, or many of them, to have a margin-left of 20 or 95px.  You would not do this.  What you would do is simply add this margin-left property to the top level container.  In this case, it's this div here.  Remove all margin-left's within this div, add this property to this div.  If you still need extra space, then you can do a margin-left on individual elements.

Reply:
This goes for all elements.  If you find yourself continuously styling each element within a div with the same margin, consider styling whatever container contains those elements intead.